example-deployment/docker/reporting.dockerfile

```diff
@@ -1,9 +1,16 @@
 # FIXME: we are importing from builder because we are installing packages related to ros on bootstrap.
-FROM rmf-web/builder
+FROM ros:foxy-ros-base-focal
```

Comment:
does the FIXME still apply?

I missed that this is a multi stage build in the previous review, so it doesn't really matter if we use the rmf builder or other images. Both the previous version and this should be fine.